runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -533,6 +556,23 @@ J9::SymbolReferenceTable::findOrCreateMethodTypeTableEntrySymbol(TR::ResolvedMet
    return symRef;
    }
 
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+TR::SymbolReference *
+J9::SymbolReferenceTable::refineInvokeCacheElementSymRefWithKnownObjectIndex(TR::ResolvedMethodSymbol * owningMethodSymbol,  TR::SymbolReference * originalSymRef, uintptr_t arrayElementRef)
+   {
+   TR_J9VMBase *fej9 = (TR_J9VMBase *)(fe());
+   TR_ASSERT(fej9->haveAccess(), "Require VM access to be acquired by caller");
+   TR::KnownObjectTable *knot = comp()->getOrCreateKnownObjectTable();
+   if (!knot) return originalSymRef;
+   TR_ResolvedJ9Method *owningMethod = static_cast<TR_ResolvedJ9Method*>(owningMethodSymbol->getResolvedMethod());
```

Comment:
owningMethod and owningMethodSymbol are not used

Reply:
this was left over from a previous revision. fixed.